global.json

```diff
@@ -1,7 +1,7 @@
 {
   "sdk": {
     "allowPrerelease": false,
-    "version": "3.1.100",
+    "version": "3.1.201",
     "rollForward": "latestPatch"
   }
```

Comment:
probably better once set to a [latestFeature](https://docs.microsoft.com/en-us/dotnet/core/tools/global-json?tabs=netcore3x#matching-rules)?

Reply:
Or just remove `global.json`

Reply:
Definitely can be overkill for many projects 👍  - for the core _Serilog_ package and a couple of other sensitive ones like this, I think the improved determinism is worth the inconvenience, but outside of this handful of packages I've tried to avoid it :-)